grunz/tests/test_file_utils.py

```diff
@@ -0,0 +1,156 @@
+import unittest
+import tempfile
+import os
+import time
+import hashlib
+from pathlib import Path
+from filecmp import cmp
+
+from grunz.file_utils.file_utils import FileUtils
+
+
+class TestFileUtils(unittest.TestCase):
+
+    def setUp(self):
+        # Creates a temporary directory for test files
+        self.test_dir = tempfile.TemporaryDirectory()
+        self.file_utils = FileUtils(Path(self.test_dir.name))
+
+    def tearDown(self):
+        # Cleans up the temporary directory and test files
+        self.test_dir.cleanup()
+
+    def test_find_files_recursively_returning_matching_files(self):
```

Comment:
Test names should try to follow an established syntax. My own personal preference is `Should_ExpectedBehavior_When_StateUnderTest`. There's a good article on that here: https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea. Hope that helps :)

Reply:
Yes makes sense to be more systematic with the names. The `Should_ExpectedBehavior_When_StateUnderTest` format looks good, I'm happy to use that. My only concern is, is it not necessary to include the name of the function being tested? 

And just to understand the format, in this case would the name be `Should_ReturnAVIFilesOnly_When_SearchAVIFiles` or something more generalised?